src/interface/src/app/navigation/navigation.component.html

```diff
@@ -4,10 +4,9 @@
       [mode]="(isHandset$ | async) ? 'over' : 'side'"
       [opened]="!(isHandset$ | async) && sidebarOpen">
     <mat-nav-list>
-      <a mat-list-item routerLink="/home">Home</a>
       <a mat-list-item routerLink="/map">Map</a>
       <div *ngIf="isLoggedIn() | async; else showLogin">
```

Comment:
Do we want to consider making isLoggedIn() an instance variable instead of a function (like isHandset$, but set in ngOnInit)? 
Or put a $ after it to show that it's an observable?